tests/collections/asr/test_asr_metrics.py

```diff
@@ -124,7 +124,12 @@ def __randomString(N):
             n2 = random.randint(1, 512)
             s1 = __randomString(n1)
             s2 = __randomString(n2)
```

Comment:
Can we force it to not generate an empty string? ie, `while s2.strip: s2 = __randomString(n2);`

Reply:
Maybe resample n2 too